getting-started/host_security_policy_examples.md

```diff
@@ -58,3 +58,51 @@ Here, we demonstrate how to define host security policies.
     * Explanation: The purpose of this policy is to audit any accesses to a critical file (i.e., '/etc/passwd'). Since we want to audit one critical file, we use matchPaths to specify the path of '/etc/passwd'.
 
     * Verification: After applying this policy, please open a new terminal (or connect to the host with a new session) and run 'sudo cat /etc/passwd'. Then, check the alert logs of KubeArmor.
+
+* System calls alerting
+  * Alert for all `unlink` syscalls
+  ```text
+  apiVersion: security.kubearmor.com/v1
+  kind: KubeArmorHostPolicy
+  metadata:
+    name: audit-all-unlink
+  spec:
+    severity: 3
+    nodeSelector:
+          matchLabels:
+            kubernetes.io/hostname: vagrant
+    syscalls:
+      matchSyscalls:
+      - syscall:
+        - unlink
+    action:
+      Audit
+  ```
+  Generated telemetry:
+
+![unlink syscall](../.gitbook/assets/hsp-unlink-syscall.png)
+
+  * Alert on all `rmdir` syscalls targeting anything in `/home/` directory and sub-directories
+  
+  ```text
+  apiVersion: security.kubearmor.com/v1
+  kind: KubeArmorHostPolicy
+  metadata:
+    name: audit-home-rmdir
+  spec:
+    severity: 3
+    nodeSelector:
+          matchLabels:
+            kubernetes.io/hostname: vagrant
+    syscalls:
+      matchPaths:
+      - syscall:
+        - rmdir
+        path: /home/
+        recursive: true
+    action:
+      Audit
+  ```
+Generated telemetry:
+
+![rmdir syscall](../.gitbook/assets/hsp-rmdir-syscall.png)
```

Comment:
How about we paste the telemetry data here and put it in a collapsible section instead of an image? Like I guess it's all text in the image anyway. WDYT?